packages/datetime/src/datePicker.tsx

```diff
@@ -116,17 +115,17 @@ export class DatePicker extends AbstractComponent<IDatePickerProps, IDatePickerS
             <div className={classNames(Classes.DATEPICKER, className)}>
                 <ReactDayPicker
                     canChangeMonth={true}
-                    captionElement={this.renderCaption()}
+                    captionElement={this.renderCaption}
                     disabledDays={this.disabledDays}
                     enableOutsideDays={true}
                     fromMonth={minDate}
-                    initialMonth={new Date(displayYear, displayMonth)}
```

Comment:
this `initialMonth` prop was changed to behave like an uncontrolled `defaultValue` and a `month` prop was added for the controlled counterpart, which is what we were using the prop for. https://github.com/gpbl/react-day-picker/issues/169